templates/shared/search/input.tmpl

```diff
@@ -0,0 +1,4 @@
+<!-- Value - value of the search field (for search results page) -->
+<!-- Disabled (optional) - if search field has to be disabled -->
+<!-- Placeholder (optional) - placeholder text to be used -->
+<input type="search" spellcheck="false" name="q" maxlength="255" placeholder="{{if .Placeholder}}{{.Placeholder}}{{else}}{{ctx.Locale.Tr "search.search"}}…{{end}}"{{if .Value}} value="{{.Value}}"{{end}}{{if .Disabled}} disabled{{else}} autofocus{{end}}>
```

Comment:
Should "autofocus" always be applied? I think in some cases the search input shouldn't grab the focus ahead.

Reply:
Also see [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus#accessibility_concerns):

> Automatically focusing a form control can confuse visually-impaired people using screen-reading technology and people with cognitive impairments. When autofocus is assigned, screen-readers "teleport" their user to the form control without warning them beforehand.

Reply:
Removed